hello-world/src/main/resources/archetype-resources/src/main/resources/HelloWorldBuilder/help-useFrench_tr.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Fransızca göster?
```

Comment:
Should the `ö` be translated to unicode?

Reply:
I only translated `.properties` files to unicode. htmls seem to render well in the browser. Try opening the the html in your browser.